index.html

```diff
@@ -10,20 +10,33 @@
 <body>
     <header class="header">
              <a href="#">Gleen</a>
-          
 
-        <nav>
-            <img src="./Images/Union.png" alt="hamburger">
-        
-            <ul class="desktop-links">
-                <li><a href="#Portfolio">Portfolio</a></li>
-                <li><a href="#About">About</a></li>
-                <li><a href="#Contact">Contact</a></li>
-            </ul>
-        </nav>
-    </header>
-
-    <section class="headline">
+             <div class="menu">
+            
+                <div id="myNav" class="overlay">
+              
+                      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+              
+                  <div class="overlay-content">
+                      <a href="#Portfolio">Portfolio</a>
+                      <a href="#About">About</a>
+                      <a href="#Contact">Contact</a>
+                  </div>
+               </div>
+              
+              <nav>
+               <span onclick="openNav()"><img src="./Images/Union.png"  alt="hamburger"></span>
+              
+               <ul class="desktop-links">
+                   <li><a href="#Portfolio">Portfolio</a></li>
```

Comment:
- [ ] [OPTIONAL] Good job on implementing the previous reviewer's requested changes :+1: however I would like to recommend you to work on this action when the user clicks (or taps) any option of the mobile menu (Example: **About or Portfolio or Contact** ) the mobile menu should disappear but in your case not. Please fix it in order to give your user a great **UX(User Experience )** :smile: 

![Screenshot from 2022-07-18 20-07-57](https://user-images.githubusercontent.com/6615394/179608282-0b673bf2-6441-49f2-aacf-8ce0d87ba7f7.png)